### PR TITLE
Partner Leaderboards

### PR DESCRIPTION
**Pull request by jameshi16**
# Changes

Added partner leaderboards. These changes ranks all Twitch users with the "Partner" into their own leaderboards

Requires #7 

# Todo list

- [x] Why does [e2f4e6c](https://github.com/vanorsigma/neuro-chat-elo/commit/e2f4e6c6c05c52885aef51a00b03080171f17538) not have any leaderboards?

### CODE REVIEW

**Comment by jameshi16** [comment]
Will be reliant on #23 xdx

File: web/src/lib/revealCards.svelte [hunk 1]
```diff
@@ -6,7 +6,7 @@
   let moveToLeaderboardsBtnTxt = 'Skip';
   export let revealMetadatas: RevealMetadata[] = [];
   export let allAnimationsDone = () => {};
-
+
```
Comment (jameshi16) [inline web/src/lib/revealCards.svelte:9]:
```suggestion

```

**Comment by jameshi16** [comment]
> This should be a separate task, but i think we should look into other ways to associate a url with a rankingInfo. Maybe a Map if possible, so the url contains the name of a leaderboard instead of an index that's subject to change.

Personally, I don't see an issue with associating it to a numerical ID. Associating it to a name has the same issues; leaderboard names are also subject to change.

**Review by Gaijutsu — APPROVED** [review]
I see what you mean about names and uids. I didn't realise that these indexes matched the currently live site.

In that case, LGTM

**Pull request merged**